kafkabp/consumer.go

```diff
@@ -7,11 +7,28 @@ import (
 	"sync/atomic"
 
 	"github.com/Shopify/sarama"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 
 	"github.com/reddit/baseplate.go/metricsbp"
 	"github.com/reddit/baseplate.go/tracing"
 )
 
+const (
+	successLabel = "success"
+)
+
+var (
+	rebalanceLabels = []string{
+		successLabel,
+	}
+
+	rebalanceCounter = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "kafka_consumer_rebalance_total",
```

Comment:
kafkabp?

Reply:
I think `kafka` is better in this case. The `bp` thing is a bit weird to me.